core/domain/rights_domain.py

```diff
@@ -29,6 +29,7 @@
 # Do not modify the definitions of CMD keys that already exist.
 CMD_CREATE_NEW = feconf.CMD_CREATE_NEW
 CMD_CHANGE_ROLE = feconf.CMD_CHANGE_ROLE
+CMD_REMOVE_ROLE = feconf.CMD_REMOVE_ROLE
```

Comment:
Prefer not doing this in the context of this PR (see other comments).

Reply:
Not done, we talked about this offline.

Reply:
Are we expecting to remove/undo these changes? (By "Not done" you mean we are going to have these changes in a separate PR, right?)

Reply:
Sorry, these changes are expected and should be part of this PR.